mkdocs.yml

```diff
@@ -105,11 +105,10 @@ plugins:
 - mkdocstrings:
     handlers:
       python:
-        import:
+        inventories:
         - https://installer.readthedocs.io/en/stable/objects.inv
-        rendering:
-          show_signature_annotations: true
         options:
+          show_signature_annotations: true
```

Comment:
it seems that `import` was renamed `inventories`: https://mkdocstrings.github.io/python/usage/#inventories

and `rendering` is merged with `options`: https://mkdocstrings.github.io/python/usage/configuration/signatures/?h=show_signature_annotations#show_signature_annotations

Reply:
Well... that is going to break a bunch of repos for sure 😂